### DISCUSSION TITLE
My account has been stolen.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#126309] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My passward has been changed.
All my Email have been illegally removed.

What should I Do?

**Comment by talhaahussain** [comment]
Hey Bryce,

To be quite honest, this is quite a serious situation and unfortunately the community cannot provide you with much assistance in the way of recovering your account.

You should look to contact GitHub support about your issue. Please see [here](https://docs.github.com/en/support/contacting-github-support) for more details. You should [create a support ticket.](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket#about-support-tickets). Quoted from this page: _You can contact GitHub Support to report account, security, and abuse issues._

I wish you the best of luck on recovering your account.

**Reply by ghost** [reply]
I have already created the support ticket.
That's a long way to wait official reply.

Thanks for your advices!

**Comment by ebndev** [comment]
Hi @Bryce1998temp,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
